README.md

```diff
@@ -18,7 +18,11 @@ The Node.js version to install. "0.10" is the default and works on all supported
 
     nodejs_install_npm_user: "{{ ansible_ssh_user }}"
 
-The user for whom the npm packages will be installed can be set here, this defaults to ansible_ssh_user
+The user for whom the npm packages will be installed can be set here, this defaults to ansible_user
```

Comment:
It could possibly be changed to drop the dependency on `ansible_user` as tasks are run as this user by default.
Often the issue is that the task is run in the context of `become: yes` (sudo) so root user perms are used.

eg.

`ansible localhost -c local -m shell -a whoami`

`ansible localhost -c local -m shell -a whoami --become`